src/main/java/com/hcl/appscan/jenkins/plugin/scanners/ScannerConstants.java

```diff
@@ -38,4 +38,6 @@ public interface ScannerConstants {
 	String FASTEST                  = "Fastest";                //$NON-NLS-1$
 	String NO_OPTIMIZATION          = "NoOptimization";         //$NON-NLS-1$
 	String TEST_OPTIMIZATION_LEVEL  = "TestOptimizationLevel";  //$NON-NLS-1$
+    	String SCAN_SPEED               = "scanSpeed";              //$NON-NLS-1$
+    	String DEEP                     = "deep";                   //$NON-NLS-1$
```

Comment:
We shouldn't need to set the scan speed to "deep" since that's the default, so this constant shouldn't be needed.

Reply:
Okay, Matt. Made the required changes.